app/screens/post_priority_picker/post_priority_picker.tsx

```diff
@@ -211,6 +212,7 @@ const PostPriorityPicker = ({
                                     type='toggle'
                                     selected={data.persistent_notifications}
                                     descriptionNumberOfLines={2}
+                                    value='persistent_notifications'
```

Comment:
I was checking the implementation of how this value is used, and something feels really off. Can you verify if this doesn't break anything? The test I would do is:

- Turn on persistent notifications
- Turn them off again (verify they get visually turned off)
- Create the post and verify the post was created without persistent notifications.

Reply:
@yasserfaraazkhan Did you check this case?